share/models/registration.py

```diff
@@ -0,0 +1,41 @@
+from model_utils import Choices
+
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext as _
+
+
+class ProviderRegistration(models.Model):
+
+    STATUS = Choices(
+        (0, 'pending', _('pending')),
+        (1, 'accepted', _('accepted')),
+        (2, 'implemented', _('implemented')),
+        (3, 'rejected', _('rejected'))
+    )
+
+    status = models.IntegerField(choices=STATUS, default=STATUS.pending)
+    submitted_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Could you make the default ordering descending on `submitted_at` and add a complementary index? The admin interface should then sort them on `submitted_at` by default.